src/server/scripts/Kalimdor/zone_bloodmyst_isle.cpp

```diff
@@ -773,7 +773,7 @@ enum FreeWebbedBloodmyst
     SPELL_FREE_WEBBED_11     = 31010
 };
 
-uint32 const SummonSpells[10] =
+std::array<uint32, 10> const SummonSpells =
```

Comment:
was this needed to use Trinity::Containers::SelectRandomContainerElement( ) ? it should work even with the old code

Reply:
no but we don't live under a rock anymore so yay for 'modern' code.

Reply:
@Ovahlord does ValidateSpellInfo() support std::array ?

Reply:
Nope. It's a list initializer which is a glorified vector so what he would have to do is copy the array content which is way too overkill for that check. Just slap a maintainer info on it to keep the enums and the array sync.

Reply:
As you said I will be not able to validate both spell list and one spell which is not inside spell list
Currently all spells are validated but manually, looks not that bad as I expected

Reply:
I'd stick to the old C-style array and just call `ValidateSpellInfo(array) && ValidateSpellInfo({EXTRA_SPELL});`